consensus/module_consensus_state_sync.go

```diff
@@ -22,21 +22,18 @@ func (m *consensusModule) GetNodeAddress() string {
 	return m.nodeAddress
 }
 
-// IsSynced implements the interface function for checking if the node is synced with the network.
-func (m *consensusModule) IsSynced() (bool, error) {
-	currentHeight := m.GetBus().GetConsensusModule().CurrentHeight()
-	readCtx, err := m.GetBus().GetPersistenceModule().NewReadContext(int64(currentHeight - 1)) // Unknown height
-	if err != nil {
-		return false, err
-	}
-	defer readCtx.Release()
-
-	maxPersistedHeight, err := readCtx.GetMaximumBlockHeight()
-	if err != nil {
-		return false, err
-	}
-
-	maxSeenHeight := m.stateSync.GetAggregatedMetadata().MaxHeight
+// TODO (#352): Implement this function, currently a placeholder.
+// commitReceivedBlocks commits the blocks received from the blocksReceived channel
+func (m *consensusModule) commitReceivedBlocks() {
+	// runs as a background process in consensus module
+	// listens on the blocksReceived channel
+	// commits the block
+}
 
-	return maxPersistedHeight == maxSeenHeight, nil
+// TODO (#352): Implement this function, currently a placeholder.
+// metadataSyncLoop periodically sends metadata requests to its peers
+func (m *consensusModule) metadataSyncLoop() {
```

Comment:
Leave a comment that his is intended to run as a background goroutine. Same for the function above.